Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:latest
```

Comment:
2 things:

1. We may want to stick to a given tag
2. We may want to use the rust image instead, so that we can run a regular `cargo build` and `cargo install`. Those images are debian based, so it should work with all the apt based instructions here. See the 1.67.1-slim-buster tag at https://registry.hub.docker.com/_/rust/ for example